spec/integration/solved_spec.rb

```diff
@@ -181,7 +181,7 @@
 
       expect(topic.public_topic_timer.status_type).to eq(TopicTimer.types[:silent_close])
 
-      expect(topic.custom_fields[DiscourseSolved::AUTO_CLOSE_TOPIC_TIMER_CUSTOM_FIELD].to_i).to eq(
+      expect(topic.custom_fields["solved_auto_close_topic_timer_id"].to_i).to eq(
```

Comment:
Curious, why not keep the constant? For readability?

Reply:
Felt like in specs it was shorter to use the string instead of the constant.